examples/web-wagmi/src/App.tsx

```diff
@@ -57,7 +58,10 @@ import { UseProfileFollowRevenue } from './revenue/UseProfileFollowRevenue';
 import { UseProfilePublicationRevenue } from './revenue/UseProfilePublicationRevenue';
 import { UsePublicationRevenue } from './revenue/UsePublicationRevenue';
 
-const { provider, webSocketProvider } = configureChains([polygon, optimism], [publicProvider()]);
```

Comment:
was this a typo? I needed `mumbai` provider for the user to be able to sign in the example app, is there a reason why we need `optimism` ?